clang/test/CheckedCRewriter/checkedregionsitypeparams.c

```diff
@@ -0,0 +1,20 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+
+struct A { 
+  int b;
+};
+
+int foo(struct A *a) { 
+  return a->b + 1;
```

Comment:
This should get rewritten to `int foo(struct A *a : itype(_Ptr<struct A>))` right? And it can/should be checked. When we rewrite a parameter or return to be an `itype`, it's when the interior of the function is safe.